cimetrics/github_pr.py

```diff
@@ -63,16 +64,40 @@ def upload_image(self, encoded_image):
         else:
             raise Exception("Failed to upload image")
 
+    def first_self_comment(self):
```

Comment:
Should this be the latest self comment instead?

Reply:
The implementation is the first, and in principle, we should only ever have one, so it should be the same? Is there a scenario/reason where's useful to take the last one instead?
All other things being equal, it's more work/execution/slower to get the last, so unless there's a use case for it, I prefer the first.

Reply:
I was thinking of the case of keeping both the old (one post per build) and new (edit one unique comment) behaviours. If the PR was started with the old behaviour and say, 5 builds were run, then the metrics file updated to the new behaviour, then only the oldest report (out of 5) would be updated.

If we are happy to remove the old behaviour (I am), then this is a not an issue,